sql/src/main/java/io/crate/analyze/OrderBy.java

```diff
@@ -160,4 +159,30 @@ public void replace(Function<? super Symbol, Symbol> replaceFunction) {
             listIt.set(replaceFunction.apply(listIt.next()));
         }
     }
+
+    @Nullable
+    public static OrderBy mergeOrderBy(OrderBy orderBy1, OrderBy orderBy2) {
```

Comment:
also maybe allow orderby to merge to be nullable and return this if its nullable